lib/index.js

```diff
@@ -176,6 +170,7 @@ class PackageJson {
   // Load data from a comment
   // /**package { "name": "foo", "version": "1.2.3", ... } **/
   fromComment (data) {
+    const parseJSON = require('json-parse-even-better-errors')
```

Comment:
I don't think having this nested `require` is providing any benefit because `./read_package.js` is already defined up top which also imports this. Can we move this up top?

Reply:
Alternatively if you're really interested in perf / load `readPackage` could also require `json-parse-even-better-errors` from within the function.